go/install/install_windows.go

```diff
@@ -255,6 +255,28 @@ func ToggleAutostart(context Context, on bool, forAutoinstallIgnored bool) error
 	return nil
 }
 
+// This is the old startup info logging. Retain it for now, but it is soon useless.
+// TODO Remove in 2021.
+func deprecatedStartupInfo(logFile *os.File) {
```

Comment:
what happens in this function on the newest version? if it looks like an error, can we instead break out early or otherwise convey that it's not an error that matters?

Reply:
The function is the same code as for the last two years. Basically it works for now, but when we start removing starting the service with the shortcut then it becomes more or less useless other than debugging installer issues.